SPHINXsys/src/shared/sphinxsys_system/sph_system.cpp

```diff
@@ -7,14 +7,14 @@
 #include "sph_system.h"
 #include "base_body.h"
 #include "particle_generator_lattice.h"
-
+//tbb_init_(number_of_threads), 
 namespace SPH
 {
 	//=================================================================================================//
 	SPHSystem::SPHSystem(Vecd lower_bound, Vecd upper_bound,
 		Real particle_spacing_ref, int number_of_threads)
 		: lower_bound_(lower_bound), upper_bound_(upper_bound),
-		tbb_init_(number_of_threads), particle_spacing_ref_(particle_spacing_ref),
```

Comment:
@Xiangyu-Hu @chuvirtonomy Is this essential for creating the class? the definition of "task_scheduler_init" has changed in the new TBB version.